### PR TITLE
⬆️ upgrade aio-pika to latest version 8.2.4

### PR DESCRIPTION
**Pull request by sanderegg**
###  Highlights on updated libraries (only updated libraries are included)

- #packages before: 4
- #packages after : 4

|#|name|before|after|upgrade|count|packages|
|-|----|------|-----|-------|-----|--------|
|   1 | aio-pika                  | 7.2.0, 6.8.0    | 8.2.4      | **MAJOR**  | 4 | director-v2⬆️🧪</br>dynamic-sidecar⬆️</br>web⬆️ |
|   2 | aiormq                    | 3.3.1, 6.2.3    | 6.4.2      | *minor*    | 4 | director-v2⬆️🧪</br>dynamic-sidecar⬆️</br>web⬆️ |
|   3 | docker                    | 5.0.3           | 6.0.0      | **MAJOR**  | 1 | dynamic-sidecar⬆️ |
|   4 | pamqp                     | 3.1.0, 2.3.0    | 3.2.1      | *minor*    | 4 | director-v2⬆️🧪</br>dynamic-sidecar⬆️</br>web⬆️ |

*Legend*: 
- ⬆️ base dependency (only services because packages are floating)
- 🧪 test dependency
- 🔧 tool dependency

### Repo-wide overview of libraries
- #reqs files parsed: 59

|#|name|versions-base|versions-test|versions-tool|
|-|----|-------------|-------------|-------------|
|   1 | aio-pika                  | 8.2.4                     | 8.2.4                     |                           |
|   2 | aioboto3                  | 9.6.0                     | 10.1.0                    |                           |
|   3 | aiobotocore               | 2.3.0, 2.3.3              | 2.4.0                     |                           |
|   4 | aiocache                  | 0.11.1                    | 0.11.1                    |                           |
|   5 | aiodebug                  | 2.3.0                     | 2.3.0                     |                           |
|   6 | aiodocker                 | 0.19.1, 0.21.0            | 0.21.0                    |                           |
|   7 | aiofiles                  | 0.8.0, 22.1.0             | 22.1.0                    |                           |
|   8 | aiohttp                   | 3.8.1, 3.8.3              | 3.8.1, 3.8.3              |                           |
|   9 | aiohttp-jinja2            | 1.5                       |                           |                           |
|  10 | aiohttp-security          | 0.4.0                     |                           |                           |
|  11 | aiohttp-session           | 2.11.0                    |                           |                           |
|  12 | aiohttp-swagger           | 1.0.16                    |                           |                           |
|  13 | aioitertools              | 0.10.0                    | 0.11.0                    |                           |
|  14 | aiopg                     | 1.3.3, 1.3.5              | 1.3.5                     |                           |
|  15 | aioredis                  | 2.0.1                     |                           |                           |
|  16 | aioresponses              |                           | 0.7.3                     |                           |
|  17 | aiormq                    | 6.4.2                     | 6.4.2                     |                           |
|  18 | aiosignal                 | 1.2.0                     | 1.2.0                     |                           |
|  19 | aiosmtplib                | 1.1.6                     |                           |                           |
|  20 | aiozipkin                 | 1.1.1                     |                           |                           |
|  21 | alembic                   | 1.8.1                     | 1.8.1                     |                           |
|  22 | anyio                     | 3.6.1, 3.6.2              | 3.6.1, 3.6.2              |                           |
|  23 | argon2-cffi               | 20.1.0                    |                           |                           |
|  24 | asgi-lifespan             |                           | 1.0.1                     |                           |
|  25 | asgiref                   | 3.5.2                     |                           |                           |
|  26 | astroid                   |                           | 2.12.11, 2.12.12          | 2.12.11                   |
|  27 | async-asgi-testclient     |                           | 1.4.11                    |                           |
|  28 | async-generator           | 1.10                      |                           |                           |
|  29 | async-timeout             | 4.0.2                     | 4.0.2                     |                           |
|  30 | asyncpg                   | 0.25.0                    |                           |                           |
|  31 | attrs                     | 21.4.0, 22.1.0            | 21.4.0, 22.1.0            |                           |
|  32 | aws-sam-translator        |                           | 1.53.0                    |                           |
|  33 | aws-xray-sdk              |                           | 2.10.0                    |                           |
|  34 | bcrypt                    | 3.2.0                     | 4.0.1                     |                           |
|  35 | beautifulsoup4            | 4.10.0                    |                           |                           |
|  36 | black                     |                           |                           | 22.10.0                   |
|  37 | bleach                    | 3.3.0                     |                           |                           |
|  38 | blosc                     | 1.10.6                    |                           |                           |
|  39 | bokeh                     | 2.4.3                     | 2.4.3                     |                           |
|  40 | boto3                     | 1.21.21, 1.24.80, 1.24.96 | 1.21.21, 1.24.59, 1.24.89 |                           |
|  41 | boto3-stubs               |                           | 1.24.89                   |                           |
|  42 | botocore                  | 1.24.21, 1.27.80, 1.27.96 | 1.24.21, 1.27.59, 1.27.89 |                           |
|  43 | botocore-stubs            | 1.27.17                   | 1.27.89                   |                           |
|  44 | build                     |                           |                           | 0.8.0                     |
|  45 | bump2version              |                           |                           | 1.0.1                     |
|  46 | certifi                   | 2022.5.18.1, 2022.6.15, 2022.9.14, 2022.9.24 | 2022.5.18.1, 2022.6.15, 2022.9.14, 2022.9.24 |                           |
|  47 | cffi                      | 1.15.0                    | 1.15.0, 1.15.1            |                           |
|  48 | cfgv                      |                           |                           | 3.3.1                     |
|  49 | cfn-lint                  |                           | 0.67.0                    |                           |
|  50 | change-case               |                           |                           | 0.5.2                     |
|  51 | charset-normalizer        | 2.0.12, 2.1.1             | 2.0.12, 2.1.1             |                           |
|  52 | click                     | 8.1.3                     | 8.1.3                     | 8.1.3                     |
|  53 | cloudpickle               | 2.0.0, 2.2.0              |                           |                           |
|  54 | codecov                   |                           | 2.1.12                    |                           |
|  55 | colorama                  | 0.4.5                     |                           |                           |
|  56 | colorlog                  |                           | 6.7.0                     |                           |
|  57 | commonmark                | 0.9.1                     |                           |                           |
|  58 | coverage                  |                           | 6.5.0                     |                           |
|  59 | coveralls                 |                           | 3.3.1                     |                           |
|  60 | cryptography              | 3.4.7, 36.0.2, 37.0.2     | 36.0.2, 38.0.1            |                           |
|  61 | cytoolz                   | 0.11.0                    |                           |                           |
|  62 | dask                      | 2022.9.2                  |                           |                           |
|  63 | dask-gateway              | 2022.10.0                 |                           |                           |
|  64 | dask-gateway-server       |                           | 2022.10.0                 |                           |
|  65 | decorator                 | 4.4.2                     |                           |                           |
|  66 | defusedxml                | 0.7.1                     |                           |                           |
|  67 | deprecated                | 1.2.13                    | 1.2.13                    |                           |
|  68 | dill                      |                           | 0.3.5.1, 0.3.6            | 0.3.5.1                   |
|  69 | distlib                   |                           |                           | 0.3.6                     |
|  70 | distributed               | 2022.9.2                  |                           |                           |
|  71 | distro                    | 1.5.0                     |                           |                           |
|  72 | dnspython                 | 2.0.0, 2.1.0, 2.2.1       | 2.2.1                     |                           |
|  73 | docker                    | 6.0.0                     | 6.0.0                     |                           |
|  74 | docker-compose            | 1.29.1                    |                           |                           |
|  75 | dockerpty                 | 0.4.1                     |                           |                           |
|  76 | docopt                    | 0.6.2                     | 0.6.2                     |                           |
|  77 | ecdsa                     |                           | 0.18.0                    |                           |
|  78 | email-validator           | 1.2.1, 1.3.0              | 1.3.0                     |                           |
|  79 | et-xmlfile                | 1.1.0                     |                           |                           |
|  80 | exceptiongroup            | 1.0.0                     | 1.0.0                     |                           |
|  81 | execnet                   |                           | 1.9.0                     |                           |
|  82 | expiringdict              | 1.2.1                     |                           |                           |
|  83 | faker                     |                           | 15.1.0, 15.1.1            |                           |
|  84 | fastapi                   | 0.85.0, 0.85.1            |                           |                           |
|  85 | fastapi-contrib           | 0.2.11                    |                           |                           |
|  86 | fastapi-pagination        | 0.10.0                    |                           |                           |
|  87 | fastjsonschema            | 2.15.3                    |                           |                           |
|  88 | filelock                  |                           |                           | 3.8.0                     |
|  89 | flaky                     |                           | 3.7.0                     |                           |
|  90 | flask                     |                           | 2.1.3                     |                           |
|  91 | flask-cors                |                           | 3.0.10                    |                           |
|  92 | frozenlist                | 1.3.0, 1.3.1              | 1.3.0, 1.3.1              |                           |
|  93 | fsspec                    | 2022.5.0, 2022.8.2        |                           |                           |
|  94 | graphql-core              |                           | 3.2.3                     |                           |
|  95 | greenlet                  | 1.1.2, 1.1.3.post0        | 1.1.2, 1.1.3.post0        |                           |
|  96 | gunicorn                  | 20.1.0                    |                           |                           |
|  97 | h11                       | 0.12.0                    | 0.12.0                    |                           |
|  98 | h2                        | 4.1.0                     |                           |                           |
|  99 | heapdict                  | 1.0.1                     |                           |                           |
| 100 | hpack                     | 4.0.0                     |                           |                           |
| 101 | httpcore                  | 0.15.0                    | 0.15.0                    |                           |
| 102 | httptools                 | 0.2.0, 0.5.0              |                           |                           |
| 103 | httpx                     | 0.23.0                    | 0.23.0                    |                           |
| 104 | hyperframe                | 6.0.1                     |                           |                           |
| 105 | hypothesis                |                           | 6.56.2                    |                           |
| 106 | icdiff                    |                           | 2.0.5                     |                           |
| 107 | identify                  |                           |                           | 2.5.6                     |
| 108 | idna                      | 2.10, 3.3, 3.4            | 2.10, 3.3, 3.4            |                           |
| 109 | importlib-metadata        | 5.0.0                     | 5.0.0                     |                           |
| 110 | iniconfig                 | 1.1.1                     | 1.1.1                     |                           |
| 111 | inotify                   |                           |                           | 0.2.10                    |
| 112 | isodate                   | 0.6.1                     |                           |                           |
| 113 | isort                     |                           | 5.10.1                    | 5.10.1                    |
| 114 | itsdangerous              | 1.1.0, 2.1.2              | 2.1.2                     |                           |
| 115 | jaeger-client             | 4.8.0                     |                           |                           |
| 116 | jinja-app-loader          | 1.0.2                     |                           |                           |
| 117 | jinja2                    | 3.1.2                     | 3.1.2                     | 3.1.2                     |
| 118 | jmespath                  | 1.0.0, 1.0.1              | 1.0.0, 1.0.1              |                           |
| 119 | jschema-to-python         |                           | 1.2.3                     |                           |
| 120 | json2html                 | 1.3.0                     |                           |                           |
| 121 | jsondiff                  | 2.0.0                     | 2.0.0                     |                           |
| 122 | jsonpatch                 |                           | 1.32                      |                           |
| 123 | jsonpickle                |                           | 2.2.0                     |                           |
| 124 | jsonpointer               |                           | 2.3                       |                           |
| 125 | jsonschema                | 3.2.0, 4.16.0             | 3.2.0, 4.16.0             |                           |
| 126 | junit-xml                 |                           | 1.9                       |                           |
| 127 | jupyter-client            | 6.1.12                    |                           |                           |
| 128 | jupyter-core              | 4.7.1                     |                           |                           |
| 129 | jupyter-server            | 1.18.1                    |                           |                           |
| 130 | jupyter-server-proxy      | 3.2.1                     |                           |                           |
| 131 | jupyterlab-pygments       | 0.1.2                     |                           |                           |
| 132 | lazy-object-proxy         | 1.7.1                     | 1.7.1                     | 1.7.1                     |
| 133 | locket                    | 1.0.0                     |                           |                           |
| 134 | lz4                       | 4.0.0                     |                           |                           |
| 135 | mako                      | 1.2.2, 1.2.3              | 1.2.2, 1.2.3              |                           |
| 136 | markupsafe                | 2.1.1                     | 2.1.1                     | 2.1.1                     |
| 137 | mccabe                    |                           | 0.7.0                     | 0.7.0                     |
| 138 | minio                     |                           | 7.0.4                     |                           |
| 139 | mistune                   | 2.0.4                     |                           |                           |
| 140 | moto                      |                           | 4.0.1, 4.0.7              |                           |
| 141 | msgpack                   | 1.0.3, 1.0.4              |                           |                           |
| 142 | multidict                 | 6.0.2                     | 6.0.2                     |                           |
| 143 | mypy-extensions           |                           |                           | 0.4.3                     |
| 144 | nbclient                  | 0.5.3                     |                           |                           |
| 145 | nbconvert                 | 7.2.1                     |                           |                           |
| 146 | nbformat                  | 5.3.0                     |                           |                           |
| 147 | nest-asyncio              | 1.5.1                     |                           |                           |
| 148 | networkx                  | 2.5.1                     | 2.8.7                     |                           |
| 149 | nodeenv                   |                           |                           | 1.7.0                     |
| 150 | nose                      |                           |                           | 1.3.7                     |
| 151 | numpy                     | 1.22.3                    | 1.23.4                    |                           |
| 152 | openapi-core              | 0.12.0                    |                           |                           |
| 153 | openapi-schema-validator  | 0.2.3                     | 0.2.3                     |                           |
| 154 | openapi-spec-validator    | 0.4.0                     | 0.4.0                     |                           |
| 155 | openpyxl                  | 3.0.9                     |                           |                           |
| 156 | opentracing               | 2.4.0                     |                           |                           |
| 157 | orjson                    | 3.7.2                     |                           |                           |
| 158 | packaging                 | 21.3                      | 21.3                      | 21.3                      |
| 159 | pamqp                     | 3.2.1                     | 3.2.1                     |                           |
| 160 | pandas                    | 1.2.4                     | 1.5.0                     |                           |
| 161 | pandocfilters             | 1.4.3                     |                           |                           |
| 162 | paramiko                  | 2.11.0                    |                           |                           |
| 163 | parfive                   | 1.5.1                     |                           |                           |
| 164 | partd                     | 1.2.0, 1.3.0              |                           |                           |
| 165 | passlib                   | 1.7.4                     | 1.7.4                     |                           |
| 166 | pathspec                  |                           |                           | 0.10.1                    |
| 167 | pbr                       |                           | 5.10.0                    |                           |
| 168 | pep517                    |                           |                           | 0.13.0                    |
| 169 | pillow                    | 9.0.1                     | 9.2.0                     |                           |
| 170 | pint                      | 0.19.2                    | 0.19.2                    |                           |
| 171 | pip-tools                 |                           |                           | 6.9.0                     |
| 172 | platformdirs              |                           | 2.5.2                     | 2.5.2                     |
| 173 | pluggy                    | 1.0.0                     | 1.0.0                     |                           |
| 174 | pprintpp                  |                           | 0.4.0                     |                           |
| 175 | pre-commit                |                           |                           | 2.20.0                    |
| 176 | prometheus-client         | 0.14.1                    |                           |                           |
| 177 | psutil                    | 5.9.1, 5.9.2              |                           |                           |
| 178 | psycopg2-binary           | 2.9.3, 2.9.4              | 2.9.3, 2.9.4              |                           |
| 179 | ptvsd                     |                           | 4.3.2                     |                           |
| 180 | ptyprocess                | 0.7.0                     |                           |                           |
| 181 | py                        |                           | 1.11.0                    |                           |
| 182 | py-cpuinfo                |                           | 8.0.0                     |                           |
| 183 | pyasn1                    |                           | 0.4.8                     |                           |
| 184 | pycparser                 | 2.20, 2.21                | 2.20, 2.21                |                           |
| 185 | pydantic                  | 1.9.0, 1.10.2             | 1.10.2                    |                           |
| 186 | pyftpdlib                 |                           | 1.5.7                     |                           |
| 187 | pygments                  | 2.9.0, 2.13.0             |                           |                           |
| 188 | pyinstrument              | 3.4.2, 4.1.1, 4.3.0       | 4.3.0                     |                           |
| 189 | pyinstrument-cext         | 0.2.4                     |                           |                           |
| 190 | pyjwt                     | 2.4.0                     |                           |                           |
| 191 | pylint                    |                           | 2.15.4, 2.15.5            | 2.15.4                    |
| 192 | pynacl                    | 1.4.0                     |                           |                           |
| 193 | pyopenssl                 |                           | 22.1.0                    |                           |
| 194 | pyparsing                 | 3.0.9                     | 3.0.9                     | 3.0.9                     |
| 195 | pyrsistent                | 0.18.1                    | 0.18.1                    |                           |
| 196 | pytest                    | 7.2.0                     | 7.1.3, 7.2.0              |                           |
| 197 | pytest-aiohttp            |                           | 1.0.4                     |                           |
| 198 | pytest-asyncio            |                           | 0.19.0, 0.20.1            |                           |
| 199 | pytest-benchmark          |                           | 3.4.1                     |                           |
| 200 | pytest-cov                |                           | 4.0.0                     |                           |
| 201 | pytest-docker             |                           | 1.0.1                     |                           |
| 202 | pytest-forked             |                           | 1.4.0                     |                           |
| 203 | pytest-icdiff             |                           | 0.6                       |                           |
| 204 | pytest-instafail          |                           | 0.4.2                     |                           |
| 205 | pytest-lazy-fixture       |                           | 0.6.3                     |                           |
| 206 | pytest-localftpserver     |                           | 1.1.4                     |                           |
| 207 | pytest-mock               |                           | 3.10.0                    |                           |
| 208 | pytest-runner             |                           | 6.0.0                     |                           |
| 209 | pytest-sugar              |                           | 0.9.5                     |                           |
| 210 | pytest-xdist              |                           | 2.5.0                     |                           |
| 211 | python-dateutil           | 2.8.1, 2.8.2              | 2.8.1, 2.8.2              |                           |
| 212 | python-dotenv             | 0.20.0, 0.21.0            | 0.20.0, 0.21.0            |                           |
| 213 | python-engineio           | 3.14.2                    |                           |                           |
| 214 | python-jose               |                           | 3.3.0                     |                           |
| 215 | python-magic              | 0.4.25                    |                           |                           |
| 216 | python-multipart          | 0.0.5                     |                           |                           |
| 217 | python-socketio           | 4.6.1                     |                           |                           |
| 218 | pytz                      | 2020.1, 2022.1            | 2022.4                    |                           |
| 219 | pyyaml                    | 5.4.1, 6.0                | 5.4.1, 6.0                | 5.4.1, 6.0                |
| 220 | pyzmq                     | 22.1.0                    |                           |                           |
| 221 | redis                     | 4.3.1, 4.3.4              | 4.3.1                     |                           |
| 222 | requests                  | 2.27.1, 2.28.1            | 2.27.1, 2.28.1            |                           |
| 223 | responses                 |                           | 0.22.0                    |                           |
| 224 | respx                     |                           | 0.20.0                    |                           |
| 225 | rfc3986                   | 1.4.0, 1.5.0              | 1.4.0, 1.5.0              |                           |
| 226 | rich                      | 12.5.1, 12.6.0            |                           |                           |
| 227 | rsa                       |                           | 4.9                       |                           |
| 228 | s3fs                      | 2022.5.0                  |                           |                           |
| 229 | s3transfer                | 0.5.2, 0.6.0              | 0.5.2, 0.6.0              |                           |
| 230 | sarif-om                  |                           | 1.0.4                     |                           |
| 231 | semantic-version          | 2.9.0                     |                           |                           |
| 232 | send2trash                | 1.7.1                     |                           |                           |
| 233 | setproctitle              | 1.2.3                     |                           |                           |
| 234 | shellingham               | 1.5.0                     |                           |                           |
| 235 | simpervisor               | 0.4                       |                           |                           |
| 236 | six                       | 1.15.0, 1.16.0            | 1.15.0, 1.16.0            |                           |
| 237 | sniffio                   | 1.2.0, 1.3.0              | 1.2.0, 1.3.0              |                           |
| 238 | sortedcontainers          | 2.4.0                     | 2.4.0                     |                           |
| 239 | soupsieve                 | 2.3.2                     |                           |                           |
| 240 | sqlalchemy                | 1.4.37, 1.4.41            | 1.4.37, 1.4.41            |                           |
| 241 | sshpubkeys                |                           | 3.3.1                     |                           |
| 242 | starlette                 | 0.20.4                    |                           |                           |
| 243 | strict-rfc3339            | 0.7                       |                           |                           |
| 244 | tblib                     | 1.7.0                     |                           |                           |
| 245 | tenacity                  | 8.0.1, 8.1.0              | 8.0.1, 8.1.0              |                           |
| 246 | termcolor                 |                           | 2.0.1                     |                           |
| 247 | terminado                 | 0.10.1                    |                           |                           |
| 248 | texttable                 | 1.6.3                     |                           |                           |
| 249 | threadloop                | 1.0.2                     |                           |                           |
| 250 | thrift                    | 0.16.0                    |                           |                           |
| 251 | tinycss2                  | 1.1.1                     |                           |                           |
| 252 | toml                      |                           | 0.10.2                    | 0.10.2                    |
| 253 | tomli                     | 2.0.1                     | 2.0.1                     | 2.0.1                     |
| 254 | tomlkit                   |                           | 0.11.5                    | 0.11.5                    |
| 255 | toolz                     | 0.11.1, 0.12.0            |                           |                           |
| 256 | tornado                   | 6.1, 6.2                  | 6.1                       |                           |
| 257 | tqdm                      | 4.64.0, 4.64.1            | 4.64.1                    |                           |
| 258 | traitlets                 | 5.1.1                     | 5.4.0                     |                           |
| 259 | twilio                    | 7.12.0                    |                           |                           |
| 260 | typer                     | 0.4.1, 0.6.1              | 0.6.1                     | 0.6.1                     |
| 261 | types-aiobotocore         | 2.3.3                     | 2.4.0.post1               |                           |
| 262 | types-aiobotocore-s3      | 2.3.3                     | 2.4.0.post1               |                           |
| 263 | types-aiofiles            |                           | 22.1.0                    |                           |
| 264 | types-awscrt              |                           | 0.14.7                    |                           |
| 265 | types-boto3               |                           | 1.0.2                     |                           |
| 266 | types-pkg-resources       |                           | 0.1.3                     |                           |
| 267 | types-pyyaml              |                           | 6.0.12                    |                           |
| 268 | types-s3transfer          |                           | 0.6.0.post4               |                           |
| 269 | types-toml                |                           | 0.10.8                    |                           |
| 270 | typing-extensions         | 4.3.0, 4.4.0              | 4.3.0, 4.4.0              | 4.3.0, 4.4.0              |
| 271 | ujson                     | 5.5.0                     |                           |                           |
| 272 | urllib3                   | 1.26.9, 1.26.11, 1.26.12  | 1.26.9, 1.26.11, 1.26.12  |                           |
| 273 | uvicorn                   | 0.15.0, 0.17.0, 0.18.3, 0.19.0 |                           |                           |
| 274 | uvloop                    | 0.16.0, 0.17.0            |                           |                           |
| 275 | virtualenv                |                           |                           | 20.16.5                   |
| 276 | watchdog                  | 2.1.5                     |                           | 2.1.9                     |
| 277 | watchfiles                | 0.18.0                    |                           |                           |
| 278 | watchgod                  | 0.8.2                     |                           |                           |
| 279 | webencodings              | 0.5.1                     |                           |                           |
| 280 | websocket-client          | 0.59.0, 1.4.1             | 0.59.0, 1.4.1             |                           |
| 281 | websockets                | 10.1, 10.2, 10.3          | 10.3                      |                           |
| 282 | werkzeug                  | 2.0.3, 2.1.2              | 2.0.3, 2.1.2              |                           |
| 283 | wheel                     |                           |                           | 0.37.1                    |
| 284 | wrapt                     | 1.14.1                    | 1.14.1                    | 1.14.1                    |
| 285 | xmltodict                 |                           | 0.13.0                    |                           |
| 286 | yarl                      | 1.5.1, 1.7.2, 1.8.1       | 1.5.1, 1.7.2, 1.8.1       |                           |
| 287 | zict                      | 2.2.0                     |                           |                           |
| 288 | zipp                      | 3.9.0                     | 3.9.0                     |                           |

### CODE REVIEW

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py [hunk 1]
```diff
@@ -73,10 +73,10 @@ def __init__(self, app: FastAPI, max_messages_to_send: int = 100) -> None:
         self._project_id: ProjectID = settings.DY_SIDECAR_PROJECT_ID
         self._node_id: NodeID = settings.DY_SIDECAR_NODE_ID
 
-        self._connection: aio_pika.Connection | None = None
-        self._channel: aio_pika.Channel | None = None
-        self._logs_exchange: aio_pika.Exchange | None = None
-        self._events_exchange: aio_pika.Exchange | None = None
+        self._connection: aio_pika.abc.AbstractConnection | None = None
+        self._channel: aio_pika.abc.AbstractChannel | None = None
```
Comment (pcrespov) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py:77]: aren't you moving all these to servicelib? Or it iwll be a separate PR?

**Review by pcrespov — APPROVED** [review]
:tada: !! Finally a new rabbit! :-)

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py (hunk 1 above)
Comment (sanderegg) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py:77]: this will be separate
